src/concrete/ml/sklearn/base.py

```diff
@@ -1690,3 +1697,358 @@ def predict_proba(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) ->
         y_logits = self.decision_function(X, fhe=fhe)
         y_proba = self.post_processing(y_logits)
         return y_proba
+
+
+# pylint: disable=invalid-name,too-many-instance-attributes
```

Comment:
`disable-next` is better, else it disables for the whole file ! If it does not work, then it's better to re-enable it after the class' definition !